action.yml

```diff
@@ -143,6 +143,13 @@ inputs:
       installed. Example: "290.0.1".
     required: false
 
+  gcloud_components:
```

Comment:
It might be more flexible if we make this `gcloud_component` and accept a string. We can do client-side validation to ensure only empty string, "alpha", and "beta" are used. However, there might be "gcloud preview" in the future. Making it a string would make for an easier future refactor. I think the default should remain as an empty string.

Reply:
First attempt....